docs/introduction.rst

```diff
@@ -1,8 +1,18 @@
 .. _Introduction to XBlocks:
 
-#############################
+#######################
 Introduction to XBlocks
-#############################
+#######################
```

Comment:
I also get an error here in the docs build, which confuses me: 
`docs/introduction.rst:5: WARNING: duplicate label introduction to xblocks, other instance in docs/introduction.rst`

Reply:
n/m I figured it out